src/core/client/stream/tabs/Comments/Comment/ModerationDropdown/RejectCommentMutation.ts

```diff
@@ -11,15 +11,36 @@ import { RejectCommentMutation as MutationTypes } from "coral-stream/__generated
 
 let clientMutationId = 0;
 
+function decrementCount(store: RecordSourceSelectorProxy, storyID: string) {
+  const storyRecord = store.get(storyID);
+  if (!storyRecord) {
+    return;
+  }
+  const commentCountsRecord = storyRecord.getLinkedRecord("commentCounts");
+  if (!commentCountsRecord) {
+    return;
+  }
+  const tagsRecord = commentCountsRecord.getLinkedRecord("tags");
+  if (tagsRecord) {
+    tagsRecord.setValue(tagsRecord.getValue("FEATURED") - 1, "FEATURED");
```

Comment:
Unless I'm misreading this, it decrements the featured count every time a comment is rejected, which isn't the case for all tagged comments (such as staff comments).

Reply:
Updated to pull back FEATURED comment counts appropriately rather than tryin to increment/decrement.

Went this route, because at this point, the feature flag is already cleared from the comment and the `store`, so we can't do a safe feature flag check before decrement.